server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -551,11 +556,35 @@ def filter(
         columns: list[str],
         step: "FilterStep",
     ) -> StepContext:
+        def _cast_to_timestamp(value: Any) -> Any:
+            if self.DIALECT != SQLDialect.GOOGLEBIGQUERY:
+                return functions.Cast(value, 'TIMESTAMP')
+
+            return functions.Cast(value, 'DATETIME')
```

Comment:
Why the distinction ?

Reply:
i was getting a weird error while trying to compare, fixed here : https://github.com/ToucanToco/weaverbird/pull/1413/commits/08503599171d1e4418722585be6b0122e0699312
Thanks pointing this out 🙏🏾